core/domain/user_services_test.py

```diff
@@ -270,6 +290,17 @@ def test_create_new_user_with_invalid_email_creates_no_user_models(self):
         self.assertEqual(user_ids_in_user_auth_details, [tmp_admin_user_id])
         self.assertEqual(user_ids_in_user_contributions, [tmp_admin_user_id])
 
+    def test_create_new_user_with_already_existing_auth_id_raises_error(self):
+        auth_id = 'someUser'
+        user_email = 'user@example.com'
+
+        user_id = user_services.create_new_user(auth_id, user_email).user_id
+        error_msg = 'User %s already exists for auth_id %s.' % (
```

Comment:
I would "group" this with the "with" clause below rather than the user_id statement above (which is still part of the setup) -- this error message is part of the test expectation.

Reply:
This comment isn't actually done, though it's marked done. The code is still the same as when I last looked at it, I think.

Reply:
Done